tests/test_ffnn.py

```diff
@@ -103,3 +106,82 @@ def test_ffnn_output(self, ffnn_config: FeedForwardCfg):
         print(
             "Equal predictions between PyTorch and NumPy",
             "Implementation of FFNN: {}% of {} predictions".format(close_percentage*100, number_of_inputs))
+
+        assert ffnn_numpy_outputs.size == np.count_nonzero(
+            np.isclose(ffnn_numpy_predict_outputs.reshape(number_of_inputs, output_size),
+                       ffnn_numpy_outputs))
+
+    def test_ffnn_predict_speedtest(self):
```

Comment:
Ist ein cooler test, aber ist mMn kein guter Unit-Test. Hier wird ja nicht die funktionalität getestet, sondern was anderes

```suggestion

    @pytest.mark.skip(reason="General Testing, not a test case of the program itself")
    def test_ffnn_predict_speedtest(self):
```